csrc/scheduler/matmul_utils.cpp

```diff
@@ -824,6 +825,16 @@ std::string getMatmulCompileTimeRejectReason(Fusion* fusion) {
   {
     for (const mma_utils::MatmulPattern& pattern : patterns) {
       Expr* op = pattern.output->definition();
+      if (device_prop->major >= 9 && op->isA<ReductionOp>()) {
```

Comment:
We could leave this mul+sum check for anther PR, I just thought of this while looking at the MatmulOp and LinearOp stuff.